packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -443,6 +471,14 @@ class DebuggingControls extends StatelessWidget {
                   'Step Out',
                 ),
               ),
+              const SizedBox(width: defaultSpacing),
+              MaterialButton(
+                onPressed: isPaused ? null : controller.pause,
+                child: const MaterialIconLabel(
+                  Icons.pause,
+                  'Pause',
```

Comment:
It's much less commonly used in a debugger but had been in the first position in the list. Pause works to pause the VM wherever it may be in execution, but generally pausing is accomplished by breakpoints / break on exceptions.